.env.example

```
@@ -1,30 +1,15 @@
-
-# Search / retrieval providers
-SEARCH_PROVIDER=none
-TAVILY_API_KEY=
-PINECONE_API_KEY=
-PINECONE_INDEX=
-
-# Screenshot enrichment
-SCREENSHOT_OCR_PROVIDER=openai
-SCREENSHOT_VISION_MODEL=gpt-4.1-mini
+PORT=3000
+DATABASE_PATH=./data/daily-timeline.db
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DATABASE_PATH key should go before the NODE_ENV key

(UnorderedKey)